examples/distributed_data_classification_examples/test_models.ipynb

```diff
@@ -0,0 +1,373 @@
+   "source": [
+    "cluster = LocalCUDACluster(rmm_async=True, rmm_pool_size=\"1GB\")\n",
+    "client = Client(cluster)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data File Paths "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input_file_path=\"/home/nfs/syurick/LLM_domain_classifier_inference/4360_results_jsonl_dir/\"\n",
+    "output_file_path = \"/raid/vjawa/output_file.parquet\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create a Classifier"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "classifier_type=\"DomainClassifier\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Reading 16 files\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 7.14 s, sys: 4.91 s, total: 12 s\n",
+      "Wall time: 7.95 s\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "\n",
+    "input_dataset = DocumentDataset.read_json(\n",
+    "    input_file_path, backend=\"cudf\", add_filename=True\n",
+    ")\n",
+    "\n",
+    "if classifier_type == \"DomainClassifier\":\n",
+    "    domain_labels = [\n",
+    "    \"Adult\",\n",
+    "    \"Arts_and_Entertainment\",\n",
+    "    \"Autos_and_Vehicles\",\n",
+    "    \"Beauty_and_Fitness\",\n",
+    "    \"Books_and_Literature\",\n",
+    "    \"Business_and_Industrial\",\n",
+    "    \"Computers_and_Electronics\",\n",
+    "    \"Finance\",\n",
+    "    \"Food_and_Drink\",\n",
+    "    \"Games\",\n",
+    "    \"Health\",\n",
+    "    \"Hobbies_and_Leisure\",\n",
+    "    \"Home_and_Garden\",\n",
+    "    \"Internet_and_Telecom\",\n",
+    "    \"Jobs_and_Education\",\n",
+    "    \"Law_and_Government\",\n",
+    "    \"News\",\n",
+    "    \"Online_Communities\",\n",
+    "    \"People_and_Society\",\n",
+    "    \"Pets_and_Animals\",\n",
+    "    \"Real_Estate\",\n",
+    "    \"Science\",\n",
+    "    \"Sensitive_Subjects\",\n",
+    "    \"Shopping\",\n",
+    "    \"Sports\",\n",
+    "    \"Travel_and_Transportation\",\n",
+    "    ]\n",
+    "    model_file_name = \"/home/nfs/syurick/LLM_domain_classifier_inference/\" + \\\n",
+    "                      \"GoogleDebertaAgree_v3b_bce_maxlen512_bs64_noRef_best.pth\"\n",
+    "    classifier = DomainClassifier(\n",
+    "        model_file_name=model_file_name,\n",
+    "        labels=domain_labels,\n",
+    "        batch_size=1024,\n",
+    "    )\n",
+    "elif classifier_type == \"QualityClassifier\":\n",
+    "    quality_labels = [\"High\", \"Medium\", \"Low\"]\n",
+    "    model_file_name = \"/home/nfs/syurick/LLM_quality_classifier_inference/\" + \\\n",
```

Comment:
This path should be changed to something more generic like `quality_classifier.pth`

Reply:
Done